test/e2e/search_test.go

```diff
@@ -710,5 +710,72 @@ var _ = ginkgo.Describe("[karmada-search] karmada search testing", ginkgo.Ordere
 				})
 			})
 		})
+		ginkgo.Describe("update cluster testing", ginkgo.Ordered, func() {
+			ginkgo.Context("cluster labeled after registry created", ginkgo.Ordered, func() {
```

Comment:
This line seems redundant.

Reply:
I didn't touch this test case, it's provided by other commiter previously. It seemed that the diff algo of github get confused, it marks this line green treated as modification of this pr, but it's not.